### PR TITLE
Added template map generator as bin script

### PR DESCRIPTION
**Pull request by RalfEggert**
Added template map generator bin script and added it as well to the composer.json file for easy installation

### CODE REVIEW

File: bin/templatemap_generator.php [hunk 1]
```diff
@@ -0,0 +1,255 @@
+#!/usr/bin/env php
+<?php
+/**
+ * Zend Framework (http://framework.zend.com/)
+ *
+ * @link      http://github.com/zendframework/zendframework for the canonical source repository
```
Comment (webimpress) [inline bin/templatemap_generator.php]: it should be now `zendframework/zend-view`

File: bin/templatemap_generator.php [hunk 2]
```diff
@@ -0,0 +1,255 @@
...
+$viewPath    = getcwd() . '/view';
+
+$rules = array(
+    'help|h'            => 'Get usage message',
+    'library|l-s'       => 'Library to parse; if none provided, assumes current directory',
+    'view|v-s'          => 'View path to parse; if none provided, assumes view as template directory',
+    'extensions|e-s'    => 'List of accepted file extensions (regex alternation: *html, phtml|tpl); default: *',
+    'output|o-s'        => 'Where to write map file; if not provided, assumes "template_map.php" in library directory',
+    'append|a'          => 'Append to map file if it exists',
+    'overwrite|w'       => 'Whether or not to overwrite existing map file',
```
Comment (webimpress) [inline bin/templatemap_generator.php]: we can remove 3 spaces in each line before `=>`

File: bin/templatemap_generator.php [hunk 3]
```diff
@@ -0,0 +1,255 @@
...
+    require __DIR__ . '/../../../autoload.php';
+} else {
+    fwrite(STDERR, "Unable to setup autoloading; aborting\n");
+    exit(2);
+}
+
+$libraryPath = getcwd();
+$viewPath    = getcwd() . '/view';
+
+$rules = array(
```
Comment (webimpress) [inline bin/templatemap_generator.php]: please use short array syntax

File: bin/templatemap_generator.php [hunk 4]
```diff
@@ -0,0 +1,255 @@
...
+    // Remove the last line as it is the end of the array, and we want to
+    // append our new templates
+    $content = file($output, FILE_IGNORE_NEW_LINES);
+    array_pop($content);
+    $content = implode(PHP_EOL, $content) . PHP_EOL;
+} else {
+    // Write mode or the file does not exists: create a new file
+    // Stupid syntax highlighters make separating < from PHP declaration necessary
+    $content = '<' . "?php" . PHP_EOL
+        . '// Generated by ZF2\'s ./bin/templatemap_generator.php'  . PHP_EOL
```
Comment (webimpress) [inline bin/templatemap_generator.php]: is it gonna be still ZF2? ;)

**Comment by RalfEggert** [comment]
@webimpress 

Here you go

https://github.com/zendframework/zend-view/pull/65/commits/6c154b0b9d5112bc5d03ec1e999715d523d2cbc5

**Pull request merged**